src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -94,15 +94,15 @@ private function configureRoutes(RoutingConfigurator $routes): void
      */
     private function getConfigDir(): string
     {
-        return $this->getProjectDir().'/config';
+        return $this->getProjectDir().'/{config}';
     }
 
     /**
      * Gets the path to the bundles configuration file.
      */
     private function getBundlesPath(): string
     {
-        return $this->getConfigDir().'/bundles.php';
+        return str_replace(['{', '}'], '', $this->getConfigDir()).'/bundles.php';
```

Comment:
let's make this simpler?
```suggestion
        return str_replace('{config}', 'config', $this->getConfigDir()).'/bundles.php';
```

Reply:
that would look for the `bundles.php` file outside the `config` dir, are you sure?

Reply:
(I updated the suggestion ;) )